code/datums/traits/negative.dm

```diff
@@ -0,0 +1,180 @@
+//predominantly negative traits
+
+
+
+/datum/trait/arachnophobia
+	name = "Arachnophobia"
+	desc = "You have a paralyzing fear of spiders."
+	value = -1
+	medical_record_text = "Patient has a persistent, genuine phobia of arachnids."
+
+/datum/trait/arachnophobia/add()
+	var/mob/living/carbon/human/H = trait_holder
+	H.gain_trauma(/datum/brain_trauma/mild/phobia, TRUE, "spiders")
+
+
+
+/datum/trait/heavy_sleeper
+	name = "Heavy Sleeper"
+	desc = "You sleep like a rock! Whenever you're put to sleep, you sleep for a little bit longer."
+	value = -1
+	mob_trait = TRAIT_HEAVY_SLEEPER
+	gain_text = "<span class='danger'>You feel sleepy.</span>"
+	lose_text = "<span class='notice'>You feel awake again.</span>"
+	medical_record_text = "Patient has abnormal sleep study results and is difficult to wake up."
+
+
+
+/datum/trait/nearsighted
+	name = "Nearsighted"
+	desc = "You are nearsighted without prescription glasses, but spawn with a pair."
+	value = -1
+	mob_trait = TRAIT_NEARSIGHT
+	gain_text = "<span class='danger'>Things far away from you start looking blurry.</span>"
+	lose_text = "<span class='notice'>You start seeing faraway things normally again.</span>"
+	medical_record_text = "Patient requires prescription glasses in order to counteract nearsightedness."
+
+/datum/trait/nearsighted/on_spawn()
+	var/mob/living/carbon/human/H = trait_holder
+	var/obj/item/clothing/glasses/regular/glasses = new(get_turf(H))
+	H.put_in_hands(glasses)
+	H.equip_to_slot(glasses, slot_glasses)
+	H.regenerate_icons() //this is to remove the inhand icon, which persists even if it's not in their hands
+
+
+
+/datum/trait/nonviolent
+	name = "Pacifist"
+	desc = "The thought of violence makes you sick. So much so, in fact, that you can't hurt anyone."
+	value = -2
+	mob_trait = TRAIT_PACIFISM
+	gain_text = "<span class='danger'>You feel repulsed by the thought of violence!</span>"
+	lose_text = "<span class='notice'>You think you can defend yourself again.</span>"
+	medical_record_text = "Patient is unusually pacifistic and cannot bring themselves to cause physical harm."
+
+/datum/trait/nonviolent/on_process()
+	if(trait_holder.mind && trait_holder.mind.antag_datums.len)
+		to_chat(trait_holder, "<span class='boldannounce'>Your antagonistic nature has caused you to renounce your pacifism.</span>")
+		qdel(src)
```

Comment:
Given that people are concerned that this might be an "antag test", perhaps you could give antagonists an action button to disable it?

Reply:
how are they actually supposed to "antag test" a pacifist

Reply:
or a pref? "Ignore traits when antagonist"

Reply:
It's not possible to "antag test" a pacifist because the player could just refuse to attack. Keeping traits as antagonist, though, is both interesting gameplay and switches it up more, so I don't see why not to keep most traits.

Reply:
I mean, I suppose they COULD refuse to attack. But it's not like we have the 6th amendment in Space Law (or anyone actually cares about spacelaw). Anyone refusing to punch someone with a tome or punch to test for non-violence, just assume they're guilty and throw them in the permabrig.

Just kidding, no one permas people, just execute them for valids.